routing/RouteSolver.py

```diff
@@ -0,0 +1,134 @@
+##################################################################################
```

Comment:
Summary of things I saw multiple times

- brackets are unnecessary in logic statements
- snake case not camel case
- when passing parameters to functions properly space them